libpod/top.go

```diff
@@ -0,0 +1,37 @@
+package libpod
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/utils"
+	"github.com/sirupsen/logrus"
+)
+
+// This can be removed is stats is merged
+var CGroupParent = "/libpod_parent"
+
+// GetContainerPids reads sysfs to obtain the pids associated with the container's cgroup
+func (c *Container) GetContainerPids() ([]string, error) {
```

Comment:
Can you add a lock/unlock and syncContainer() to the beginning to make sure the container still exists when we call this